src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -83,6 +84,15 @@ public void storeMessage(Message message) {
     return messages;
   }
 
+// <<<<<<< statspage-azan
```

Comment:
nit: seems leftover from merge, you can remove this comment and the one below.

Reply:
Removed. I thought we can keep it for tracking changes later.